sdk/tests/conformance/offscreencanvas/context-creation-worker.html

```diff
@@ -39,21 +39,22 @@
 <script>
 description("This test ensures that the WebGL context can be created on an OffscreenCanvas.");
 
-if (!window.OffscreenCanvas)
+if (!window.OffscreenCanvas) {
+  testSkipped("OffscreenCanvas is not supported.");
```

Comment:
Here and throughout, please don't use testSkipped(). It prevents a 100% pass rate of the conformance suite when it's run via the webgl-conformance-tests.html harness.

Instead, please either just use debug() to report the fact that the test was skipped, or use testPassed().